apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -93,8 +93,8 @@ const Ticket = new GQLListSchema('Ticket', {
             schemaDoc: 'When status of the ticket was changed to completed',
             type: DateTimeUtc,
         },
-        lastCommentAt: {
-            schemaDoc: 'Last comment time in ticket',
+        lastResidentCommentAt: {
```

Comment:
Why we need this field? we already have TicketCommentsTime

Reply:
this field is needed to filter tickets by the date of the last comment of a resident

Reply:
It’s better to create two separate field for that

Reply:
done